webapps/frontend/ui/admin/client/scripts/pages/execution-metrics.js

```diff
@@ -346,6 +346,7 @@ const Controller = [
           }
 
           $scope.monthlyMetrics = angular.copy(monthlyMetricsArray);
+          $scope.monthlyMetrics = $scope.monthlyMetrics.slice(0, 12);
           updateMonthlyChart(monthlyMetricsArray.reverse());
```

Comment:
🤔 I think the sliced variable should be passed into `#updateMonthlyChart` as well.